src/team-content.js

```diff
@@ -109,6 +109,20 @@ const teamContent = [
     avatar: 'https://user-images.githubusercontent.com/586552/46751001-5c8a0300-cc87-11e8-84bb-9024b03df74f.jpg',
     gif: 'https://user-images.githubusercontent.com/334891/50363219-bab14d00-0538-11e9-9595-48b3c45358ab.gif',
     alumni: false
+  },
+  {
+    name: 'Zain Khoja',
```

Comment:
It looks like new team members usually add themselves to the top of this list :)

Reply:
This is right! Go ahead and move yourself on up above :)